laplacian-smoothing/README.md

```diff
@@ -0,0 +1 @@
+# Laplacian smoothing Radium plugin example
```

Comment:
Tell me more

Reply:
> Regarding Plugins creating a UI widget, it is more secure to create it when requested.
This is out of scope of this PR. The old plugin example code was not modified nor refactored except to be consistant with the Radium version of PR #462. 
> Why not using the `_ra` suffix?
cf above
> It is not radium, but would it be interesting to use the same formatting?
cf above

Reply:
This is a todo